pom.xml

```diff
@@ -48,6 +48,7 @@
         <module>scimono-client</module>
         <module>scimono-server</module>
         <module>scimono-examples</module>
+        <module>scim-system-compliant-tests</module>
```

Comment:
The tests aren't scim compliant on their own. Rename to something more accurate. Maybe scimono-compliance-tests